src/main/java/com/savvato/tribeapp/controllers/AttributesAPIController.java

```diff
@@ -47,9 +53,15 @@ public ResponseEntity<Boolean> applyPhraseToUser(@RequestBody @Valid AttributesR
             rtn = true;
         }
 
-        if (rtn)
+        if (rtn) {
```

Comment:
This logic is not correct yet... If the phrase is valid, we should apply the phrase to the user, create a notification, and return a True ResponseEntity. Else if it is not valid, also create a notification telling the user so, and return a False ResponseEntity (not a BAD_REQUEST, just False).